lib/editor/components/timetable/TimetableHeader.js

```diff
@@ -251,22 +257,35 @@ export default class TimetableHeader extends Component<Props> {
               overlay={
                 <Popover
                   id='popover-advanced-settings'
-                  title='Advanced settings'>
+                  title={this.messages('advancedSettings')}>
+                  {/* Allow seconds in offset and trip series tools */}
+                  <Checkbox
+                    defaultChecked={useSecondsInOffset}
+                    onChange={toggleUseSeconds}
+                  >
+                    <OverlayTrigger
+                      placement='bottom'
+                      overlay={
+                        <Tooltip id='use-seconds-in-offset'>
+                          {this.messages('useSecondsToolTip')}
+                        </Tooltip>
+                      }>
+                      <small>{this.messages('useSeconds')}</small>
+                    </OverlayTrigger>
+                  </Checkbox>
                   {/* Hide departures checkbox */}
                   {activePattern && !activePattern.useFrequency
                     ? <Checkbox
-                      value={hideDepartureTimes}
+                      defaultChecked={hideDepartureTimes}
```

Comment:
(Just noting that hiding departure times seems to hide the arrivals, visually, instead (all the remaining content is gray, which denote departures).)

Reply:
Great point. I think I may sort that out in a separate PR. I guess we just warn the user that if they're different don't use that feature: 
<img width="229" alt="image" src="https://github.com/ibi-group/datatools-ui/assets/63798641/b4ebf5b8-dd57-466a-9416-8ff59f65e14b">
It also messes up all the visuals of the timetable. Honestly haven't heard of anyone using the feature